doc/source/release-notes/Percona-Server-5.5.57-38.9.rst

```diff
@@ -0,0 +1,67 @@
+.. rn:: 5.5.57-38.9
+
+==========================
+Percona Server 5.5.57-38.9
+==========================
+
+Percona is glad to announce the release of Percona Server 5.5.57-38.9
+on August 17, 2017.
+Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.55-38.9/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.5.57
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-57.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.5.57-38.9 is now the current stable release in the 5.5 series.
+All software developed by Percona is open-source and free.
+Details of this release can be found
+in the `5.5.57-38.9 milestone on Launchpad
+<https://launchpad.net/percona-server/+milestone/5.5.57-38.9>`_.
+
+.. note:: Red Hat Enterprise Linux 5 (including CentOS 5 and other derivatives),
+   Ubuntu 12.04 and older versions are no longer supported by Percona software.
+
+New Features
+============
+
+* Added support and packages for Debian 9 (stretch).
```

Comment:
Please consult https://github.com/percona/percona-server/pull/1851/files and https://github.com/percona/percona-server/pull/1854/files for feature and bug bug descriptions